src/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+#include directories helps to include header files
```

Comment:
I'm not crazy about adding a non-source file to a directory called "src". Is there any way this file can be moved, so that we can keep the `src` directory pristine?